src/resources/util/truncatedTextField.jsx

```diff
@@ -0,0 +1,9 @@
+import { useRecordContext } from 'react-admin';
+
+export const TruncatedTextField = ({ record }) => {
```

Comment:
sorry that was my mistake, but the parameter should just be called `source`. Then you don't need the record context and you can basically revert the previous commit

Otherwise this field would only work if you used it on the text column.

Reply:
i don't know what you mean if a remvoe the record context its not working

Reply:
Okay you do need the record context, but otherwise I can only repeat what I said. The source parameter tells you which column we are looking at, but in your current implementation you always use `record.text`.

I'll just push a commit and show you.

Reply:
okay thanks